Makefile

```diff
@@ -251,15 +251,8 @@ bin/:
 
 build-static: tidy bin/spire-server-static bin/spire-agent-static bin/k8s-workload-registrar-static bin/oidc-discovery-provider-static
 
-define binary_rule_static
-.PHONY: $1
-$1: | go-check bin/
-	@echo Building $1...
-	$(E)$(go_path) CGO_ENABLED=0 go build $$(go_flags) -ldflags $$(go_ldflags) -o $1 $2
-
-endef
 # https://7thzero.com/blog/golang-w-sqlite3-docker-scratch-image
```

Comment:
I'm not sure if it still makes sense to keep this reference here or not ... it feels old but there's no year on the post

Reply:
I'd be ok removing it 🤷